src/DoctrineWriter.php

```diff
@@ -262,47 +287,50 @@ protected function disableLogging()
      */
     protected function reEnableLogging()
     {
-        $config = $this->entityManager->getConnection()->getConfiguration();
+        //TODO: do we need to add support for MongoDB logging?
+        if (!($this->objectManager instanceof \Doctrine\ORM\EntityManager)) return;
```

Comment:
Alternatively we could create a OdmReader and an OrmReader which extends a common class, so that specific features are only present when they are necessary. @smilesrg @Baachi what do you think?